pyproject.toml

```
@@ -29,7 +29,7 @@ requires-python = ">=3.9"
 dependencies = [
   "pulpcore>=3.49.0,<3.85",
   "pkginfo>=1.10.0,<1.13.0",
-  "bandersnatch>=6.3,<7.0",  # Anything >6.3 requires Python 3.10+
+  "bandersnatch~=6.3.0",  # Anything >6.3 requires Python 3.10+
```

Should this be
```suggestion
  "bandersnatch>=6.3.0,<6.4",  # Anything >6.3 requires Python 3.10+
```
then?

---

Aren't these identical?

---

Yes, and no.
They mean the same to python, but dependabot treats them differently.